commit b2a2361e9f8b23364819a010f01a51634ab00066
Author: mjw <mjw@a5019735-40e9-0310-863c-91ae7b9d1cf9>
Date:   Wed Sep 9 13:40:23 2015 +0000

    libvex_test: Use arm64_[di]MinLine_lg2_szB values that make libvex happy.
    
    git-svn-id: svn://svn.valgrind.org/valgrind/trunk@15647 a5019735-40e9-0310-863c-91ae7b9d1cf9

diff --git a/none/tests/libvex_test.c b/none/tests/libvex_test.c
index 5718e54..57184c0 100644
--- a/none/tests/libvex_test.c
+++ b/none/tests/libvex_test.c
@@ -221,6 +221,10 @@ int main(int argc, char **argv)
    // Use some values that makes AMD64 happy.
    vta.abiinfo_both.guest_stack_redzone_size = 128;
 
+   // Use some values that makes ARM64 happy.
+   vta.archinfo_guest.arm64_dMinLine_lg2_szB = 6;
+   vta.archinfo_guest.arm64_iMinLine_lg2_szB = 6;
+
    // Prepare first for a translation where guest == host
    // We will translate the get_guest_arch function
    vta.arch_guest                 = guest_arch;
